app/views/homes/index.html.erb

```diff
@@ -2,4 +2,6 @@
 <p>
   Modify me in `app/views/homes/index.html.erb` or change the root directive in
   `config/routes.rb`
+
+  This is our static page
 </p>
```

Comment:
This file should only be whats on line 1 ideally.  We can remove 2-7

Reply:
This was resolved in merge conflict resolution.